sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+from sympy import Basic, Mul, degree, Symbol, expand, cancel, Expr
+from sympy.core.evalf import EvalfMixin
+from sympy.core.numbers import Integer
+from sympy.core.sympify import sympify, _sympify
+
+__all__ = ['TransferFunction',]
+
+
+class TransferFunction(Basic, EvalfMixin):
+    """TransferFunction(num, den, var)
+
+    A class for representing Transfer Functions. This class is used to represent
+    LTI (Linear, time-invariant) systems in transfer function form. The arguments
+    are ``num``, ``den``, and ``var``, where ``num`` and ``den`` are numerator and
+    denominator of the ``TransferFunction`` respectively, and the third argument is
+    a variable used to anchor the symbols. ``num`` and ``den`` can be either
+    sympy expressions or numbers, whereas ``var`` has to be a Symbol. This class is a
+    candidate for a full symbolic model rather than just being a simple solver.
+
+    Parameters
+    ==========
+
+    num : Numerator
```

Comment:
```suggestion
    num : Expr
```
The thing after the colon is the acceptable types.

Reply:
It should be 
```
name : type
    description
```
for all parameters, and for Returns